py/dml/types.py

```diff
@@ -582,7 +655,13 @@ def declaration(self, var):
         else:
             return 'uint8 ' + self.const_str + var + '[' + str(self.bytes) + ']'
 
-class TLong(IntegerType):
+class ArchDependentIntegerType(IntegerType):
+    '''Integer types whose definition and/or properties are architecture
```

Comment:
Singular, `Integer type whose ... is ...`, because an instance represents one such type.

Reply:
... _though i forgot `is_arch_dependent`, oops_